.gitlab-ci.yml

```diff
@@ -245,21 +245,27 @@ linux64-build:
     BUILD_TARGET: linux64
 
 linux64_cxx20-build:
-  extends: .build-template
+  extends:
+    - .build-template
+    - .skip-in-fast-mode-template
   needs:
     - x86_64-pc-linux-gnu-debug
   variables:
     BUILD_TARGET: linux64_cxx20
 
 linux64_sqlite-build:
-  extends: .build-template
+  extends:
+    - .build-template
+    - .skip-in-fast-mode-template
   needs:
     - x86_64-pc-linux-gnu-debug
   variables:
     BUILD_TARGET: linux64_sqlite
 
 linux64_fuzz-build:
-  extends: .build-template
+  extends:
+    - .build-template
+    - .skip-in-fast-mode-template
```

Comment:
I think `fuzz` build is important for `fast mode` because usually it catches high percentage of all failures.

There is almost hundred of C++ source and any changes in class/structure interfaces, any noticeable big refactoring or big changes almost always cause a fail during fuzz build.

I vote for keeping `linux64_fuzz-build` in fast-mode

Reply:
hmmm... but we don't really run `src/test/fuzz/fuzz` anywhere as far as I can tell, we just compile it

Reply:
just on this week PRs that failed for `fuzz` related code (compile, not run!):
 - https://github.com/dashpay/dash/pull/5310 - https://gitlab.com/dashpay/dash/-/jobs/4087550129
 - https://github.com/dashpay/dash/pull/5306 - https://gitlab.com/dashpay/dash/-/pipelines/832877324
 -